### ISSUE TITLE
update cinder webhook handler to reflect multiple queues

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#21741]
We currently assume all cinder-managed jobs are in a single queue per env, but now that's not the case - there are multiple queues (per entity type, and split down further between themes and non-themes) plus a escalation and copyright queues.  We need to handle them all, while still ignoring the the amo-managed jobs - and the MozSoc decisions we also get too.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-667)

**Comment by diox** [comment]
Reports will go to:
- Collections -> `amo-collections`
- Extensions/Dictionaries -> `amo-listings`
- Ratings -> `amo-ratings`
- Themes -> `amo-themes`
- Users -> `amo-users`

The `amo-` prefix is different on dev (`amo-dev-`) and stage (`amo-stage-`). dev & stage still share the same Cinder instance.

Decisions will be processed from _any_ queue. Notably, moderators will be able to escalate something into a different global queue for all Mozilla projects and execute the relevant policies there (as long as those are policies supported by AMO).

**Comment by diox** [comment]
Doc with all the details about policies, entities and actions: https://docs.google.com/document/d/1b5YbQSrBq_K38RIslOaNABLaPe0NBHYJccCL7Q9X3Cc/edit

At the moment staging policies have been recreated and look broken so it can't be tested properly, we're on it.

**Comment by ioanarusiczki** [comment]
@diox 

About the queues:
- besides  addon infringement, collections, listings, ratings, themes and users they're other queues such as Escalations or deprecated content, in total I see 20 queues but I've focused on those mentioned.  I've sent reports from AMO and I received them as expected, in the correct queue. 
- Data is  loading, no issues with that today. 
-  For User, the middle panel is loading extensions metadata. I've counted around 20 when user has lots of extensions (my case 1000+)
- Collections have "Comments" field where collection notes from AMO is sent (I didn't notice it until now)
- rantings and developer replies - middle section looks good, dev replies are displayed together with the rating

Policies:
- Privacy Policy, Other and Image policies don't work yet. This needs a followup because it's unclear what's it going to be for now 
- Ignore/Approve and Acceptable Use policies work as expected 

Followups: https://github.com/mozilla/addons-server/issues/21830 and https://github.com/mozilla/addons-server/issues/21831
